helm/prometheus-operator-app/README.md.gotmpl

```diff
@@ -0,0 +1,84 @@
+# {{ .Name }}
```

Comment:
Nice job, but this file seems to be more of a contributing guide rather than a README. Beware that README shows up in happa also.

Please rename this to something like `CONTRIBUTING.md`

Reply:
Do we still need this `README.md.gotmpl` file ?